ZPerl/Libs/LibClassicCasterino/LibClassicCasterino.lua

```diff
@@ -2,7 +2,7 @@
 LibClassicCasterino
 Author: d87
 --]================]
-if WOW_PROJECT_ID ~= WOW_PROJECT_CLASSIC then return end
+if WOW_PROJECT_ID >= WOW_PROJECT_CLASSIC then return end
```

Comment:
Wrong logic.  This is supposed to return if not classic.  You just made it return IF classic.

I would go with
```
if WOW_PROJECT_ID == WOW_PROJECT_MAINLINE then return end
```